### DISCUSSION TITLE
Contribution - Correction manuel d'installation

### DISCUSSION THREAD

**Discussion by victorgyt** [dbarzin/mercator#558] [General]
Bonjour,
Je profite pour apporter une petite correction pour les commandes cités pour la génération des test de régression. 

Pour installer le plugin dusk, il manque le paramètre "-u" après sudo (qui devrait rend la commande comme ce-ci "sudo -u www-data php artisan dusk:chrome-driver"). 

Merci 

Cordialement

**Comment by dbarzin** [comment]
Fait ! Merci, Didier
